asset/js/setup.js

```diff
@@ -32,6 +32,34 @@ function setupWithSearch(siteData) {
   VueStrap.installEvents(vm);
 }
 
+/**
+ *   site-nav-btn function to toggle itself and site navigation elements.
+ */
+// eslint-disable-next-line no-unused-vars
+function siteNavToggle(siteNavBtn) {
+  siteNavBtn.classList.toggle('morph');
+  document.getElementById('site-nav').classList.toggle('open');
+  document.getElementById('site-nav-btn-wrap').classList.toggle('open');
+}
+
 jQuery.getJSON(`${baseUrl}/siteData.json`)
   .then(siteData => setupWithSearch(siteData))
   .catch(() => setup());
+
+/**
+ * Creates event listener for all dropdown-btns in page.
+ */
+Array.prototype.forEach.call(document.getElementsByClassName('dropdown-btn'),
+                             element => element.addEventListener('click', function () {
+                               this.classList.toggle('dropdown-open');
+                               const dropdownContent = this.nextElementSibling;
+                               if (dropdownContent.style.maxHeight === '0px'
+                                 || !dropdownContent.style.maxHeight) {
+                                 dropdownContent.style.transition = 'max-height 0.25s ease-in';
```

Comment:
As discussed, these can actually be put inside the .css file instead of using inline styles. Then you just need to modify the sibling's class instead of the sibling's inline style.